doc/howto/storage_backup.md

```diff
@@ -0,0 +1,109 @@
+# How to back up storage volumes
+
+There are two ways to back up your storage volumes: with snapshots or with export files.
+Snapshots are cheap and easy to create, while export files are more reliable and can also be used to restore a volume into a different storage pool.
```

Comment:
Depending on storage pool type snapshots are not cheap or easy.

Reply:
Also snapshots are taken on the same disk and machine whereas proper backups can be stored on a different disk or (ideally for a proper backup) machine.

Reply:
What is the benefit of snapshots then?

Reply:
> Depending on storage pool type snapshots are not cheap or quick (do we have a summary page where each storage pool drivers pros and cons are explained, perhaps we could link to that?)

What is the benefit of snapshots then?

We have https://linuxcontainers.org/lxd/docs/latest/reference/storage_drivers/#feature-comparison

Reply:
Yep that will help to link to to help  flag which storage engines have efficient snapshots. The benefit of snapshots is that (on pools that have quick ones) one can quickly restore to a previous state if needed. Useful for testing changes that may be problematic.

Reply:
https://github.com/lxc/lxd/blob/master/lxd/storage/drivers/driver_lvm_volumes.go#L1186